sql-plugin/src/main/scala/com/nvidia/spark/rapids/MetaUtils.scala

```diff
@@ -132,12 +132,18 @@ object MetaUtils extends Arm {
 
   /**
    * Build a TableMeta message for a degenerate table (zero columns or rows)
-   * @param tableId the ID to use for this table
    * @param batch the degenerate columnar batch
    * @return heap-based flatbuffer message
    */
-  def buildDegenerateTableMeta(tableId: Int, batch: ColumnarBatch): TableMeta = {
+  def buildDegenerateTableMeta(batch: ColumnarBatch): TableMeta = {
     require(batch.numRows == 0 || batch.numCols == 0, "batch not degenerate")
+    if (batch.numCols > 0) {
```

Comment:
I am not sure how a degenerate table can be compressed. Maybe it's a byproduct of all tables getting compressed. Perhaps a comment might help?